compiler/src/compiler/string_to_rcst.rs

```diff
@@ -1040,6 +1041,28 @@ mod parse {
                 }
             ))
         );
+        // foo T
+        //
+        // bar = 5
```

Comment:
In the string, there are three newline characters

```suggestion
        // foo T
        //
        //
        // bar = 5
```